en/mkdocs.yml

```diff
@@ -81,16 +81,19 @@ nav:
        - 'Introduction': learn/consumer-authentication.md
        - 'Configuration': learn/consumer-authentication-configuration.md
     - 'Metadata Cache Management': learn/metadata-cache-management.md
+    - 'Pushed Authorization Endpoint': learn/pushed-authorization-endpoint.md
     - 'Administration API': learn/administration-api.md
+    - 'Arrangement Management API': learn/arrangement-management-api.md
```

Comment:
```suggestion
    - 'CDR Arrangement Management API': learn/arrangement-management-api.md
```

Reply:
Shall we use **CDR** Arrangement Management, https://consumerdatastandardsaustralia.github.io/standards/#security-endpoints?